MiniApp/Classes/Display/MiniAppExternalWebViewController.swift

```diff
@@ -53,6 +54,11 @@ class MiniAppExternalWebViewController: UIViewController {
             self.webView.load(URLRequest(url: url))
         }
     }
+
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(true)
```

Comment:
you should use the animated parameter to call the super

Reply:
Done